openqa_review/browser.py

```diff
@@ -112,52 +115,40 @@ def get_page(self, url, as_json=False, cache=True):
         return content
 
     def _get(self, url, as_json=False):  # pragma: no cover
-        last_error = ""
-        for i in range(1, 7):
+        retries = Retry(total=7, backoff_factor=2, status_forcelist=[429, 500, 502, 503, 504])
+        http = requests.Session()
+        http.mount("https://", HTTPAdapter(max_retries=retries))
```

Comment:
what does the "https://" mean at this point? Why does it have to be specified? Does it prevent the use of http://?

Reply:
It means we have an adapter for the gven scheme. You could have different ones per scheme depending on requirements, see https://docs.python-requests.org/en/master/user/advanced/#transport-adapters
So yes. I need to add that as well here. Thank you for noticing!

Reply:
I suggest something like the following. Some lines below there is already `server_name = urlparse(url).netloc` which you can pull up here, for example:

```suggestion
        u = urlparse(url)
        http.mount(u.scheme, HTTPAdapter(max_retries=retries))
        …
        server_name = u.netloc
        
```